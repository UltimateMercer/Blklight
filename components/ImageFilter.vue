<template>
  <svg
    xmlns="http://www.w3.org/2000/svg"
    class="svg-filters"
    style="display: block;"
  >
    <filter
      id="cyberpunk-t"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0.01 0.28 0.01 0" />
        <feFuncG type="table" tableValues="0 0.04 0.66 1" />
        <feFuncB type="table" tableValues="0.02 1 0.96 0.5" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-u"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0 0.28 0.01" />
        <feFuncG type="table" tableValues="1 0.04 0.66" />
        <feFuncB type="table" tableValues="0.5 1 0.96" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-y"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        result="grayscale"
        values="1 0 0 0 0
                          1 0 0 0 0
                          1 0 0 0 0
                          0 0 0 1 0"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0.01 0.28 0.01" />
        <feFuncG type="table" tableValues="0 0.04 0.66" />
        <feFuncB type="table" tableValues="0.02 1 0.96" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
    </filter>
    <filter
      id="cyberpunk-ult"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0.07 0.39 0 0" />
        <feFuncG type="table" tableValues="0.07 0 0.59 1" />
        <feFuncB type="table" tableValues="0.07 1 1 0.78" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult2"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0.88 0.2 0 0 0.1" />
        <feFuncG type="table" tableValues="0 0 0 0.69 1" />
        <feFuncB type="table" tableValues="1 1 0 1 0.39" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult3"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0.88 0.2 0" />
        <feFuncG type="table" tableValues="0 0 0.69" />
        <feFuncB type="table" tableValues="1 1 1" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult4"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0 1" />
        <feFuncG type="table" tableValues="0.2 0" />
        <feFuncB type="table" tableValues="1 0.88" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult5"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="1 0 0 0.1" />
        <feFuncG type="table" tableValues="0 0.2 0 1" />
        <feFuncB type="table" tableValues="0.88 1 0 0.39" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult6"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0 0.39 0" />
        <feFuncG type="table" tableValues="0 0.1 0.98" />
        <feFuncB type="table" tableValues="0 1 1" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult7"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0 0.39 0 1" />
        <feFuncG type="table" tableValues="0 0.1 0.98 0.98" />
        <feFuncB type="table" tableValues="0 1 1 0.1" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
    <filter
      id="cyberpunk-ult8"
      x="-10%"
      y="-10%"
      width="120%"
      height="120%"
      filterUnits="objectBoundingBox"
      primitiveUnits="userSpaceOnUse"
      color-interpolation-filters="sRGB"
    >
      <feColorMatrix
        type="matrix"
        values=".33 .33 .33 0 0
            .33 .33 .33 0 0
            .33 .33 .33 0 0
            0 0 0 1 0"
        in="SourceGraphic"
        result="colormatrix"
      />
      <feComponentTransfer in="colormatrix" result="componentTransfer">
        <feFuncR type="table" tableValues="0 0.39 0 1" />
        <feFuncG type="table" tableValues="0 0 0.78 1" />
        <feFuncB type="table" tableValues="0 1 1 0" />
        <feFuncA type="table" tableValues="0 1" />
      </feComponentTransfer>
      <feBlend
        mode="normal"
        in="componentTransfer"
        in2="SourceGraphic"
        result="blend"
      />
    </filter>
  </svg>
</template>
<script>
export default {};
</script>
<style lang="scss">
.hover-card {
  &-uv,
  &-teal {
    &:hover {
      .cyberpunk-effect {
        transition: 1s ease-in;
        filter: url("#cyberpunk-ult8");
      }
    }
  }
}
</style>
