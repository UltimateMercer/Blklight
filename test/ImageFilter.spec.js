import { shallowMount } from "@vue/test-utils";
import ImageFilter from "@/components/ImageFilter.vue";

const factory = () => {
  return shallowMount(ImageFilter, {});
};

describe("ImageFilter", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.find("svg").vm).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
