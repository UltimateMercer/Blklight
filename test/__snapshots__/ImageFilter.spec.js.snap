// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageFilter renders properly 1`] = `
"<svg xmlns=\\"http://www.w3.org/2000/svg\\" style=\\"display: block; width: auto !important; height: 0px;\\">
  <filter id=\\"cyberpunk-ult\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0.39 0 1\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0 0.78 1\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 1 0\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"cyberpunk-ult_alt\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0 0.39 0.78 0 1\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0 0 0 0.78 1\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0 1 1 1 0\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"cyberpunk-hot\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0.39 0.92 1\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0 0.12 1\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 0.2 0\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"cyberpunk-ultw\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0.39 0 1 1\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0 0.78 1 1\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 1 0 1\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"blue-red\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.14 0.92\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.13 0\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.49 0.2\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"pink-blue\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0.1 1 0.92\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0.39 0 0.92\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 0.59 0.92\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"blue-pink\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0 0.91\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0.32 0.02\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 0.56\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"blue-pink2\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 0 0.91 0.92\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0.32 0.02 0.92\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 1 0.56 0.92\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"pink\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0.07 1 0.92\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0.07 0 0.92\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"0.07 0.78 0.92\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
  <filter id=\\"purple-pink\\" x=\\"-10%\\" y=\\"-10%\\" width=\\"120%\\" height=\\"120%\\" filterUnits=\\"objectBoundingBox\\" primitiveUnits=\\"userSpaceOnUse\\" color-interpolation-filters=\\"sRGB\\">
    <feColorMatrix type=\\"matrix\\" values=\\".33 .33 .33 0 0
          .33 .33 .33 0 0
          .33 .33 .33 0 0
          0 0 0 1 0\\" in=\\"SourceGraphic\\" result=\\"colormatrix\\"></feColorMatrix>
    <feComponentTransfer in=\\"colormatrix\\" result=\\"componentTransfer\\">
      <feFuncR type=\\"table\\" tableValues=\\"0 1 1\\"></feFuncR>
      <feFuncG type=\\"table\\" tableValues=\\"0 0 1\\"></feFuncG>
      <feFuncB type=\\"table\\" tableValues=\\"1 0.98 1\\"></feFuncB>
      <feFuncA type=\\"table\\" tableValues=\\"0 1\\"></feFuncA>
    </feComponentTransfer>
    <feBlend mode=\\"normal\\" in=\\"componentTransfer\\" in2=\\"SourceGraphic\\" result=\\"blend\\"></feBlend>
  </filter>
</svg>"
`;
